### PR TITLE
fix(portrait): portrait story not loading in storybook - FE-2789

### LINKED ISSUE #2917: [Portrait] docs page does not load
### Current behaviour

image

### Expected behaviour

image

### Reproducible example
https://carbon.sage.com/?path=/docs/portrait--default
### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| carbon-react     | 20.0.1
| carbon-factory   |
| react-scripts    | 
| React            |
| Browser          | Chrome
| npm              |
| Operating System | MacOS

### PR DESCRIPTION
**Pull request by mkrds**
BREAKING CHANGE: removes classic theme support from portrait

### Proposed behaviour
Portrait story should work

### Current behaviour
It doesn't.

### Checklist

- [ ] Screenshots are included in the PR
- [x] Carbon implementation and Design System documentation are congruent
- [ ] All themes are supported
- [x] Commits follow our style guide
- [x] Unit tests added or updated
- [ ] Cypress automation tests added or updated
- [x] Storybook added or updated
- [ ] Typescript `d.ts` file added or updated

### Additional context
Closes #2917 

### Testing instructions
Storybook -> Portrait

### CODE REVIEW

File: src/components/portrait/portrait.spec.js [hunk 1]
```diff
@@ -242,7 +207,7 @@ describe('PortraitComponent', () => {
     const testFail = element => renderFindTypeFail(element, PortraitInitials);
 
     it('renders initials when supplied with initials but no Gravatar or src', () => {
-      renderClassic(<Portrait initials='AB' size='medium' />);
+      // renderClassic(<Portrait initials='AB' size='medium' />);
```
Comment (edleeks87) [inline src/components/portrait/portrait.spec.js:210]: Missed this commented out line

File: src/components/portrait/__definition__.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ const definition = new Definition('portrait', Portrait, {
  `,
   propOptions: {
     size: OptionsHelper.sizesFull,
-    shape: OptionsHelper.shapesVaried
+    shape: OptionsHelper.shapesPortrait
```
Comment (edleeks87) [inline src/components/portrait/__definition__.js:17]: Think this file can be deleted as it was used for the old demo site

File: src/components/portrait/portrait-gravatar.component.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ PortraitGravatar.propTypes = {
   /** The size of the Gravatar. */
   size: PropTypes.oneOf([...OptionsHelper.sizesFull, ...OptionsHelper.sizesPortrait]).isRequired,
   /** The shape of the Gravatar. */
-  shape: PropTypes.oneOf([...OptionsHelper.shapesVaried, ...OptionsHelper.shapesPortrait]),
+  shape: PropTypes.oneOf(OptionsHelper.shapesPortrait),
```
Comment (damianOfKyan) [inline src/components/portrait/portrait-gravatar.component.js:50]: this should be an array

File: src/components/portrait/portrait-initials.component.js [hunk 4]
```diff
@@ -37,13 +36,10 @@ class PortraitInitials extends React.Component {
...
     let context = canvas.getContext('2d');
+
     let { dimensions } = getSizeParams(theme, size);
 
-    if (!isClassic(theme)) {
-      // For non-Classic themes, reduce the size by 2 pixels to
-      // allow for the CSS border around the canvas <img> element.
-      dimensions -= 2;
-    }
+    dimensions -= 2;
```
Comment (damianOfKyan) [inline src/components/portrait/portrait-initials.component.js:42]: you can move to to the line above and do

`let { dimensions } = getSizeParams(theme, size) - 2;`

File: src/components/portrait/portrait-initials.component.js [hunk 5]
```diff
@@ -116,7 +112,7 @@ PortraitInitials.propTypes = {
   /** Use a dark background. */
   darkBackground: PropTypes.bool,
   /** The shape of the Portrait. */
-  shape: PropTypes.oneOf([...OptionsHelper.shapesVaried, ...OptionsHelper.shapesPortrait]),
+  shape: PropTypes.oneOf(OptionsHelper.shapesPortrait),
```
Comment (damianOfKyan) [inline src/components/portrait/portrait-initials.component.js:115]: i think array is needed for correct prop checking? 

```suggestion
  shape: PropTypes.oneOf([OptionsHelper.shapesPortrait]),
```

File: src/components/portrait/portrait-size.config.js [hunk 6]
```diff
@@ -12,18 +12,6 @@ const sizeParams = {
...
-  'medium-small': { dimensions: 40,  iconDimensions: 24,   iconPadding: 12 },
-  medium:         { dimensions: 60,  iconDimensions: 32,   iconPadding: 14 },
-  'medium-large': { dimensions: 70,  iconDimensions: 32,   iconPadding: 18 },
-  large:          { dimensions: 100, iconDimensions: 48,   iconPadding: 25 },
-  'extra-large':  { dimensions: 120, iconDimensions: 64,   iconPadding: 27 }
-};
-
-/* eslint-enable key-spacing, no-multi-spaces */
-
 export default function getSizeParams(theme, size) {
```
Comment (damianOfKyan) [inline src/components/portrait/portrait-size.config.js:15]: can we get rid of that function entirely maybe?

File: src/components/portrait/portrait-gravatar.component.js (hunk 3 above)
Comment (mkrds) [reply src/components/portrait/portrait-gravatar.component.js:50]: OptionsHelper.shapesPortait is an array

File: src/components/portrait/portrait-initials.component.js (hunk 5 above)
Comment (mkrds) [reply src/components/portrait/portrait-initials.component.js:115]: It is an array already

**Pull request merged**